linopy/model.py

```diff
@@ -357,6 +357,37 @@ def check_force_dim_names(self, ds: DataArray | Dataset) -> None:
         else:
             return
 
+    def check_valid_dim_names(self, ds: DataArray | Dataset) -> None:
+        """
+        Ensure that the added data does not lead to a naming conflict.
+
+        Parameters
+        ----------
+        model : linopy.Model
+        ds : xr.DataArray/Variable/LinearExpression
+            Data that should be added to the model.
+
+        Raises
+        ------
+        ValueError
+            If broadcasted data leads to unsupported dimension names.
+
+        Returns
+        -------
+        None.
+        """
+        unsupported_dim_names = ["labels", "coeffs", "vars", "sign", "rhs"]
+        contains_unsupported_dim_names = any(
+            dim in unsupported_dim_names for dim in list(ds.dims)
+        )
+        if contains_unsupported_dim_names:
```

Comment:
```suggestion
        if any(dim in unsupported_dim_names for dim in ds.dims):
```
Maybe a little more condensed?